content/reference/admin/private_locations/private_packages/index.md

```diff
@@ -9,71 +9,134 @@ weight: 22033
 
 # Introduction
 
-Using Private Locations allows you to run your simulations in your private cloud. Now, what about your simulations' packages? They may hold private information, so you may want to also store them in your private cloud.
+Private Locations facilitate running simulations in your dedicated cloud environment. 
+Ensure secure storage of sensitive simulation packages in your private cloud. 
 
-Your Control Plane has a private repository feature, allowing you to manage your simulation packages privately, you just have to enable it.
+The Control Plane offers a private repository; enable it for confidential package management!
 
 # Private packages
 
-## Infrastructure
+To create private packages, utilize our Gatling Enterprise Cloud packages API and upload them through a 
+plugin configured with the Control Plane's private package repository and a Configurer public API Token.
+
+When initiating a Gatling run, the Control Plane generates a temporary signed link to allow the download of the private package from the load generators.
 
 {{< img src="private_packages_general_architecture.png" alt="Infrastructure schema" >}}
 
 {{< alert info >}}
 Private Packages are based on Private Locations. You can not use them with managed locations.
 {{< /alert >}}
 
-## Installation
+## Infrastructure
 
-First step: follow the control plane installation guide for private locations.
+Currently, private packages can be integrated with either AWS S3 or GCP Cloud Storage repositories.3
 
 {{< alert info >}}
-Before going further, ensure that your AWS S3 bucket is ready to hold your packages.
+Before going further, ensure that your repository is ready to hold your packages.
```

Comment:
Add a "see Control Plane repository section for detailed permissions requirements" with a link to the anchor, I think

Reply:
Permissions refer to the control plane more than the repository here.